context/brand-context/springernature/README.md

```diff
@@ -1,3 +1,89 @@
+## Fonts
+
+We are using Merriweather as a variable font for both `serif` and `sans-serif`.
+
+Browsers who support variable fonts do support the `woff2` format. We therefore
+do not need to serve other formats.
+
+Browsers who do not support variable fonts, will fallback to the system fonts
+specified in the `$context--font-family-serif` and `$context--font-family-sans`
+SASS variables after respectively `'Merriweather'` and `'Merriweather Sans'`.
+
+So far our `Merriweather` font instance only includes the [Weight variation
+axis](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Fonts/Variable_Fonts_Guide#weight).  
+At the time of writing `Merriweather` supports the additional [Width
+variation](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Fonts/Variable_Fonts_Guide#width) and [Optical Size variation](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Fonts/Variable_Fonts_Guide#optical_size) as well.  
+If at any point in the future there is a need for these variations, you will
+need to start over the font generation process as described below, leaving in
+the variations you need. **Beware** though, the font file weight will increase
+with the amount of variations you add.
+At the time of writeing `Merriweather` only supports the [Weight variation
+axis](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Fonts/Variable_Fonts_Guide#weight).  
+It is worth noting that we also optimised the available `font-weight` range at
+the font generation step to only span from `300` to `700`.  
+If at any point in the future there is a need for a different range, you will
+need to start over the font generation process as described below, specifying
+the new range. **Beware** though, the font file weight will increase the broader
+the range is.
+
```

Comment:
You start by saying "At the time of writing `Merriweather` supports the additional..."

but then say

"At the time of writing `Merriweather` only supports the..."

Should one of those be Merriweather Sans?